### PR TITLE
Release Microsoft.sql 2021 11 01 preview

### PR DESCRIPTION
**Pull request by ericshape**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

1. Most of the diffs are not real changes, only changing the order: in our script, we started to sort the content in the auto-generated manifest file
2. We ignore [MISSING_APIS_IN_DEFAULT_TAG](https://github.com/Azure/avocado/blob/master/README.md#MISSING_APIS_IN_DEFAULT_TAG)
3. We ignore ❌ [R4009 - RequiredReadOnlySystemData](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/openapi-authoring-automated-guidelines.md#R4009) because SQL does not fully support SystemData property. This exception has been approved by ARM in 2021.

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  5. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  6. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  7. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [x] Removing API(s) in a stable version
https://github.com/Azure/azure-rest-api-specs/pull/18005#issuecomment-1055926162
We remove the legacy "ReplicaitonLinks_unlink" API. It is a redundant and dummy API; no user uses it. 

- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by ericshape** [comment]
SQL does not support systemData until now. The feature will be completed TBD. 
❌ [R4009 - RequiredReadOnlySystemData](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/openapi-authoring-automated-guidelines.md#R4009)

**Comment by ericshape** [comment]
This error is expected. We need to ignore it.

We remove the dummy ReplicationLinks_Unlnk API endpoint. This API did not in use for many years. 

Rule | Message
-- | --
❌ MISSING_APIS_IN_DEFAULT_TAG | The default tag does not contain all APIs in this RP. Please make sure the missing API swaggers are in the default tag.readme: specification/sql/resource-manager/readme.mdjson: Microsoft.Sql/stable/2014-04-01/replicationLinks.json

Rule	Message
❌ [MISSING_APIS_IN_DEFAULT_TAG](https://github.com/Azure/avocado/blob/master/README.md#MISSING_APIS_IN_DEFAULT_TAG)	The default tag does not contain all APIs in this RP. Please make sure the missing API swaggers are in the default tag.
readme: [specification/sql/resource-manager/readme.md](https://github.com/Azure/azure-rest-api-specs/blob/9485ed5c97318a1deb8ccfa7d76c240f2156dbc7/specification/sql/resource-manager/readme.md)
json: [Microsoft.Sql/stable/2014-04-01/replicationLinks.json](https://github.com/Azure/azure-rest-api-specs/blob/9485ed5c97318a1deb8ccfa7d76c240f2156dbc7/specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/replicationLinks.json)

**Comment by j5lim** [comment]
>         "administratorLoginPassword": {

Can you annotate it with x-ms-secret?

[https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret)

---
Refers to: specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/ManagedInstances.json:854 in b3060af. [](commit_id = b3060afadf4db5da007fa3ea6ed95a3e27f0d828, deletion_comment = False)

**Comment by j5lim** [comment]
>         "administratorLoginPassword": {

Can you annotate it with x-ms-secret?

[https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret)

---
Refers to: specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/Servers.json:897 in b3060af. [](commit_id = b3060afadf4db5da007fa3ea6ed95a3e27f0d828, deletion_comment = False)

**Comment by j5lim** [comment]
>         "publicBlob": {

Is this not a secret?

---
Refers to: specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/ServerTrustCertificates.json:240 in b3060af. [](commit_id = b3060afadf4db5da007fa3ea6ed95a3e27f0d828, deletion_comment = False)

**Comment by j5lim** [comment]
>         "hubDatabasePassword": {

Can you annotate it with x-ms-secret?

[https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret)

---
Refers to: specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/SyncGroups.json:947 in b3060af. [](commit_id = b3060afadf4db5da007fa3ea6ed95a3e27f0d828, deletion_comment = False)

**Comment by j5lim** [comment]
> {

Where is this example used? I couldn't locate the api..

---
Refers to: specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/examples/CreateDwDatabaseCrossSubscriptionPITR.json:1 in b3060af. [](commit_id = b3060afadf4db5da007fa3ea6ed95a3e27f0d828, deletion_comment = False)

File: specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/Databases.json [hunk 1]
```diff
@@ -191,6 +191,15 @@
           },
           "Creates a database with ledger on.": {
             "$ref": "./examples/CreateDatabaseLedger.json"
+          },
+          "Creates a data warehouse database as a cross-subscription restore from a restore point of an existing database.": {
+            "$ref": "./examples/CreateDwDatabaseCrossSubscriptionPITR.json"
```
Comment (ericshape) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/Databases.json:196]: @j5lim here is for CreateDwDatabaseCrossSubscriptionPITR.json file

**Comment by ericshape** [comment]
> > {
> 
> Where is this example used? I couldn't locate the api..
> 
> Refers to: specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/examples/CreateDwDatabaseCrossSubscriptionPITR.json:1 in [b3060af](https://github.com/Azure/azure-rest-api-specs/commit/b3060afadf4db5da007fa3ea6ed95a3e27f0d828). [](commit_id = [b3060af](https://github.com/Azure/azure-rest-api-specs/commit/b3060afadf4db5da007fa3ea6ed95a3e27f0d828), deletion_comment = False)

here is the usage: https://github.com/Azure/azure-rest-api-specs/pull/18005#discussion_r820031854

**Comment by ericshape** [comment]
> > ```
> >     "hubDatabasePassword": {
> > ```
> 
> Can you annotate it with x-ms-secret?
> 
> https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret
> 
> Refers to: specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/SyncGroups.json:947 in [b3060af](https://github.com/Azure/azure-rest-api-specs/commit/b3060afadf4db5da007fa3ea6ed95a3e27f0d828). [](commit_id = [b3060af](https://github.com/Azure/azure-rest-api-specs/commit/b3060afadf4db5da007fa3ea6ed95a3e27f0d828), deletion_comment = False)

@j5lim addressed your x-ms-secret comments in 7d092f0b534e9e23ed93ce0793cb8ddf08be6168

**Review by j5lim — APPROVED** [review]
Approved with suggestions.

**Comment by raych1** [comment]
@ericshape , can you resolve conflicts?

**Comment by ericshape** [comment]
@raych1 do you know why the pipeline is not triggered?

**Comment by ericshape** [comment]
we ignore these errors due to the existing legacy specs and example files cannot be deleted. 



Rule | Message
-- | --
❌ UNREFERENCED_JSON_FILE | The example JSON file is not referenced from the swagger file.readme: specification/sql/resource-manager/readme.mdjson: stable/2014-04-01-legacy/examples/ReplicationLinkDelete.json
❌ UNREFERENCED_JSON_FILE | The example JSON file is not referenced from the swagger file.readme: specification/sql/resource-manager/readme.mdjson: stable/2014-04-01-legacy/examples/ReplicationLinkFailover.json
❌ UNREFERENCED_JSON_FILE | The example JSON file is not referenced from the swagger file.readme: specification/sql/resource-manager/readme.mdjson: stable/2014-04-01-legacy/examples/ReplicationLinkUnlink.json
❌ UNREFERENCED_JSON_FILE | The swagger JSON file is not referenced from the readme file.readme: specification/sql/resource-manager/readme.mdjson: Microsoft.Sql/stable/2014-04-01-legacy/replicationLinks_legacy.json
❌ MISSING_APIS_IN_DEFAULT_TAG | The default tag does not contain all APIs in this RP. Please make sure the missing API swaggers are in the default tag.readme: specification/sql/resource-manager/readme.mdjson: Microsoft.Sql/stable/2014-04-01-legacy/replicationLinks_legacy.json

Rule	Message
❌ [UNREFERENCED_JSON_FILE](https://github.com/Azure/avocado/blob/master/README.md#UNREFERENCED_JSON_FILE)	The example JSON file is not referenced from the swagger file.
readme: [specification/sql/resource-manager/readme.md](https://github.com/Azure/azure-rest-api-specs/blob/c19a51bc9a65f9d1de592449ce26c4810cb76ff6/specification/sql/resource-manager/readme.md)
json: [stable/2014-04-01-legacy/examples/ReplicationLinkDelete.json](https://github.com/Azure/azure-rest-api-specs/blob/c19a51bc9a65f9d1de592449ce26c4810cb76ff6/specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01-legacy/examples/ReplicationLinkDelete.json)
❌ [UNREFERENCED_JSON_FILE](https://github.com/Azure/avocado/blob/master/README.md#UNREFERENCED_JSON_FILE)	The example JSON file is not referenced from the swagger file.
readme: [specification/sql/resource-manager/readme.md](https://github.com/Azure/azure-rest-api-specs/blob/c19a51bc9a65f9d1de592449ce26c4810cb76ff6/specification/sql/resource-manager/readme.md)
json: [stable/2014-04-01-legacy/examples/ReplicationLinkFailover.json](https://github.com/Azure/azure-rest-api-specs/blob/c19a51bc9a65f9d1de592449ce26c4810cb76ff6/specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01-legacy/examples/ReplicationLinkFailover.json)
❌ [UNREFERENCED_JSON_FILE](https://github.com/Azure/avocado/blob/master/README.md#UNREFERENCED_JSON_FILE)	The example JSON file is not referenced from the swagger file.
readme: [specification/sql/resource-manager/readme.md](https://github.com/Azure/azure-rest-api-specs/blob/c19a51bc9a65f9d1de592449ce26c4810cb76ff6/specification/sql/resource-manager/readme.md)
json: [stable/2014-04-01-legacy/examples/ReplicationLinkUnlink.json](https://github.com/Azure/azure-rest-api-specs/blob/c19a51bc9a65f9d1de592449ce26c4810cb76ff6/specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01-legacy/examples/ReplicationLinkUnlink.json)
❌ [UNREFERENCED_JSON_FILE](https://github.com/Azure/avocado/blob/master/README.md#UNREFERENCED_JSON_FILE)	The swagger JSON file is not referenced from the readme file.
readme: [specification/sql/resource-manager/readme.md](https://github.com/Azure/azure-rest-api-specs/blob/c19a51bc9a65f9d1de592449ce26c4810cb76ff6/specification/sql/resource-manager/readme.md)
json: [Microsoft.Sql/stable/2014-04-01-legacy/replicationLinks_legacy.json](https://github.com/Azure/azure-rest-api-specs/blob/c19a51bc9a65f9d1de592449ce26c4810cb76ff6/specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01-legacy/replicationLinks_legacy.json)
❌ [MISSING_APIS_IN_DEFAULT_TAG](https://github.com/Azure/avocado/blob/master/README.md#MISSING_APIS_IN_DEFAULT_TAG)	The default tag does not contain all APIs in this RP. Please make sure the missing API swaggers are in the default tag.
readme: [specification/sql/resource-manager/readme.md](https://github.com/Azure/azure-rest-api-specs/blob/c19a51bc9a65f9d1de592449ce26c4810cb76ff6/specification/sql/resource-manager/readme.md)
json: [Microsoft.Sql/stable/2014-04-01-legacy/replicationLinks_legacy.json](https://github.com/Azure/azure-rest-api-specs/blob/c19a51bc9a65f9d1de592449ce26c4810cb76ff6/specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01-legacy/replicationLinks_legacy.json)

**Review by raych1 — APPROVED** [review]

**Pull request merged**